sdks/wasm/src/library_mono.js

```diff
@@ -192,18 +193,29 @@ var MonoSupportLib = {
 				var fetch_promise = fetch_file_cb (locateFile(deploy_prefix + "/" + file_name));
 
 				fetch_promise.then (function (response) {
-					if (!response.ok)
-						throw "failed to load '" + file_name + "'";
-					loaded_files.push (response.url);
-					return response ['arrayBuffer'] ();
+					if (!response.ok) {
+						// If it's a 404 on a .pdb, we don't want to block the app from starting up.
+          					// We'll just skip that file and continue (though the 404 is logged in the console).
+						if (response.status === 404 && file_name.match(/\.pdb$/) && MONO.mono_wasm_ignore_pdb_load_errors) {
```

Comment:
Might be better was `.endswith` if that exists, and case insensitive. But ok either way.
Dubious to make such policy decisions though.
No perfect answer.

Reply:
The code was done this way to match current functionality in Blazor.

We can change this to use something like:

```
file_name.substring(fileName.lastIndexOf('.') + 1) === "pdb";
```

But since it is to match the current Blazor funtionality I do not think it is necessary at this time.

Reply:
Case insensitivity might be added.  Did not think about that but we can address that in another PR if necessary.